06.wc/my_wc.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'optparse'
+
+def main
+  options = ARGV.getopts('l')
+  contents = read_contents
+  display_lines_words_bytes(options, contents)
+end
+
+def display_lines_words_bytes(options, contents)
```

Comment:
```suggestion
def display_lines_words_bytes(contents, options)
```

optionはオマケ？なので、後ろに持ってくることが多いですね。